src/Schema/Directives/RenameDirective.php

```diff
@@ -41,6 +41,24 @@ public static function definition(): string
      * @throws \Nuwave\Lighthouse\Exceptions\DirectiveException
      */
     public function resolveField(FieldValue $fieldValue): FieldValue
+    {
+        $attribute = $this->getAttribute();
+
+        return $fieldValue->setResolver(
+            function ($rootValue) use ($attribute) {
+                return data_get($rootValue, $attribute);
+            }
+        );
+    }
+
+    /**
+     * Retrieves the attribute argument for the directive.
+     *
+     * @return string
+     *
+     * @throws DirectiveException
+     */
+    public function getAttribute(): string
     {
         $attribute = $this->directiveArgValue('attribute');
 
```

Comment:
While we are at it, let's throw a `DefinitionException` here instead of a `DirectiveException`